packages/@react-spectrum/datepicker/stories/DatePicker.stories.tsx

```diff
@@ -11,84 +11,243 @@
  */
 
 import {action} from '@storybook/addon-actions';
+import {CalendarDate, CalendarDateTime, parseAbsolute, parseAbsoluteToLocal, parseDate, parseDateTime, parseZonedDateTime, toZoned} from '@internationalized/date';
 import {DatePicker} from '../';
+import {Flex} from '@react-spectrum/layout';
+import {Item, Picker, Section} from '@react-spectrum/picker';
+import {Provider} from '@react-spectrum/provider';
 import React from 'react';
 import {storiesOf} from '@storybook/react';
+import {useLocale} from '@react-aria/i18n';
 
 const BlockDecorator = storyFn => <div>{storyFn()}</div>;
 
-storiesOf('DatePicker', module)
+storiesOf('Date and Time/DatePicker', module)
   .addDecorator(BlockDecorator)
   .add(
     'default',
```

Comment:
Yeah if you enter a value that's too big for the field, it starts over basically. What did you expect would happen?

Reply:
starting over is fine, it's just weird that the other two advance to the next segment and this doesn't have anywhere to advance to.. Maybe it would be better to shift tab focus to the next element since you've technically completed a four digit year?